src/org/thoughtcrime/securesms/OnSwipeTouchListener.java

```diff
@@ -0,0 +1,58 @@
+package org.thoughtcrime.securesms;
+import android.content.Context;
+import android.view.GestureDetector;
+import android.view.GestureDetector.SimpleOnGestureListener;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.View.OnTouchListener;
+
+/**
+ * Created by shawnelbaz on 2018-02-09.
+ */
+
+/**
+ * Detects left and right swipes across a view.
```

Comment:
no comments + redundant comment

Reply:
this is a redundant class, we were using to implement a feature differently using swipes @g-harel

Reply:
redundant, can be removed